camptix.js

```diff
@@ -228,4 +228,13 @@ var docCookies={getItem:function(e){return decodeURIComponent(document.cookie.re
 
 	$( document ).ready( lazyLoad.init )
 
+    /**
+	 * Use select2 for dropdown
+      */
+    $( document ).ready( function loadSelect2Country() {
```

Comment:
use tabs instead of spaces
can use our `phpcs` with our `phpcs.xml.dist` file to find and fix automatically
git precommit hook to run phpcs and phpmd automatically